### DISCUSSION TITLE
Change post privacy settings after posting

### DISCUSSION THREAD

**Discussion by imnotjack91** [pixelfed/pixelfed-rn#125] [Ideas]
I am unable to edit the posts public settings after it’s been posted through the app.

**Comment by HackHerz** [comment]
Same for me. I change my profile from private to public but my posts cannot be changed :/

**Comment by renegadespork** [comment]
I would also like this feature. Changing a post from public to private or vice-versa without having to delete and repost would be really useful.

**Comment by beedell-roke** [comment]
I have posted [`pixelfed/issues/6211`](https://github.com/pixelfed/pixelfed/issues/6211#issue-3553892727) for this.
